Api/ride.py

```diff
@@ -1,35 +1,39 @@
 import json
 from flask_restful import Resource, Api
-from flask import request
+from flask import request, make_response, jsonify
 from marshmallow import Schema, fields
-# from User import driver_details
+from models.ride_models import Rrequest, DriverOffer
 from Api.schema_v import rideschema
 from models.ride_models import Rrequest, DriverOffer
 
+
 ride_Offers = []
 # where offers made by driver are stored
 request_details = {}
 # where passenger request details is stored
 ride_Requests =[]
 # where passanger ride requests details is stored
+
 class RideRequest(Resource):
     # passanger posts a ride request
     def post(self):
         postRequest = request.get_json()
+
         # validate using schema
         data,errors =rideschema.load(postRequest)
```

Comment:
Space missing after the assignment operator. This will make your code more readable.